src/polkaswap-connect-with-google.md

```diff
@@ -0,0 +1,99 @@
+
+::: info
+You can use the JSON file to import an account on Polkaswap or another wallet.
+:::
+
+![](.gitbook/assets/google-login-8.png)
+
+Success! You have created a SORA account to use on Polkaswap using Google!
+
+![](.gitbook/assets/google-importing-4.png)
+
+To access Polkaswap using this account, click on the account’s name, then input the account password you set in the last step.
+
+**_You are now connected through Google with style and freedom!_**
+
+## Importing an Existing Google Account
+
+1. With the JSON file of the account you want to import easy to reach, navigate to the Account section on Polkaswap and click the Switch account button.
+
+![](.gitbook/assets/google-importing-1.png)
+
+2. Click on Google and sign in to the account you want to use. You will find a screen with options to create or import an account if you have already signed in. Select the **Import Account** option.
+
+![](.gitbook/assets/google-importing-2.png)
+
+3. Here you can drag and drop the JSON file for your account, and there are also tutorials to help you export your JSON file from [Fearless
+   Wallet](https://wiki.fearlesswallet.io/accounts/walkthrough/exporting-and-importing-a-wallet-using-a-json-file), [Polkadot JS](https://support.polkadot.network/support/solutions/articles/65000177677-how-to-export-your-json-backup-file), and [Subwallet.](https://docs.subwallet.app/extension-user-guide/export-and-backup-an-account)
+
+![](.gitbook/assets/google-importing-3.png)
+
+4. After you drop your JSON file, Polkaswap will ask you for the account password to be able to import your account.
+
+![](.gitbook/assets/google-importing-4.png)
+
+5. After you input your account password, it will be available to select within the “Connect With Google” screen.
+
+![](.gitbook/assets/google-importing-5.png)
+
+## Making Transactions with a Google Account
+
+After your Google account has been selected as the active account, your balances will be available in the Wallet view.
+
+![](.gitbook/assets/google-transaction-1.png)
+
+1. When swapping with your Google account, the process remains the same. Select the assets to swap, confirm, and confirm the fees.
+
+![](.gitbook/assets/google-transaction-2.png)
+
+2. The Polkaswap interface will request your password to sign the transaction for your Google account, keeping it as safe as any dApp signer. You can also set Polkaswap to remember your password for fifteen minutes in _[Substrate Style](create-an-address.md#via-polkadot-js-browser-plugin)_.
+
+![](.gitbook/assets/google-transaction-3.png)
+
+The same will apply to liquidity provision transactions, both double (_pooling_) and single-sided (_staking_)—as well as Bridge transactions and claiming rewards.
+
+### Read More: [Liquidity Operations on Polkaswap](https://wiki.sora.org/provide-liquidity-to-xyk-pools-polkaswap.html), [Swaps on Polkaswap](https://wiki.sora.org/swap-polkaswap.html)
```

Comment:
```suggestion
## Learn More
- [Liquidity Operations on Polkaswap](https://wiki.sora.org/provide-liquidity-to-xyk-pools-polkaswap.html)
- [Swaps on Polkaswap](https://wiki.sora.org/swap-polkaswap.html)
```

Reply:
@outoftardis it seems like this is outdated and I cannot commit

Reply:
Also, I will be using this format to address [189](https://github.com/sora-xor/sora-docs/issues/189) 
Thank you.